treeherder/model/models.py

```diff
@@ -817,6 +818,26 @@ def update_autoclassification_bug(self, bug_number):
             return classification.set_bug(bug_number)
 
 
+class TaskclusterMetadata(models.Model):
```

Comment:
I'm presuming this is a separate table since:
* there are currently non-taskcluster jobs
* it avoids a costly alter table on the jobs table

In a world where we only have taskcluster jobs (and where we're ignoring the initial cost of a jobs alter table), does a separate table still make sense?